### PR TITLE
Terminology corrections

### PR DESCRIPTION
**Pull request by meker12**
Revised content that used the following terms incorrectly:
_allow_, _below_, _machine_, _information on_ per Rackspace style guide:
http://rackerlabs.github.io/docs-rackspace/style-guide/terminology-guidelines.html
Partially addresses issues found during [Quality Audit](https://github.com/rackerlabs/docs-cloud-monitoring/issues/8).

@cyrichardson Please review this PR when you have a chance. Thanks!

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-e4ddb774ea/docs/rackspace-monitoring/v1/)

**Comment by cyrichardson** [comment]
@meker12 Will review 1st thing tomorrow!

File: api-docs/tech-ref-info/alert-triggers-and-alarms.rst [hunk 1]
```diff
@@ -375,9 +375,9 @@ policies and their trade-offs are described next.
 	zones report resource CRITICAL and the previous alert state was OK.
 
 	The ``ALL`` policy is the most accurate, but is also prone to failure in
-	significant failure scenarios. If a network partition between our
-	internal data centers happens, the alert could be delayed due to the
-	election process. In this case, a machine has to be marked down,
+	significant failure scenarios. If a network partition hapens between our
```
Comment (cyrichardson) [inline api-docs/tech-ref-info/alert-triggers-and-alarms.rst]: hapens to happens

File: api-docs/tech-ref-info/alert-triggers-and-alarms.rst [hunk 2]
```diff
@@ -433,14 +433,13 @@ such as bytes_in on an network interface, this will give you the
...
 
 .. note::
 
-   	Notice the order of the two statements below, since it executes
-   	sequentially it is important to be most specific as the first matched
-   	condition wins. This is true for all conditions, it is commonly
-   	exposed in statements like this.
+   	Notice the order of the following two statements.  Because the function runs
+   	sequentially, you must specify the most specific match condition first.
+   	This is true for all conditions, it is commonly exposed in statements like this.
```
Comment (cyrichardson) [inline api-docs/tech-ref-info/alert-triggers-and-alarms.rst]: all conditions and is commonly

**Comment by cyrichardson** [comment]
@meker12 - 2 small nits reported inline

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-df476669d8/docs/rackspace-monitoring/v1/)

**Pull request merged**